model_compression_toolkit/qat/keras/quantizer/ste_rounding/symmetric_ste.py

```diff
@@ -132,20 +138,42 @@ def __call__(self,
 
         _min = self.quantizer_parameters[FQ_MIN]
         _max = self.quantizer_parameters[FQ_MAX]
-        if self.quantization_config.weights_channels_axis:
+        if self.channel_axis:
             if self.perm_vec:
                 inputs = tf.transpose(inputs, perm=self.perm_vec)
             q_tensor = tf.quantization.fake_quant_with_min_max_vars_per_channel(inputs, _min, _max,
-                                                                                num_bits=self.quantization_config.weights_n_bits)
+                                                                                num_bits=self.num_bits)
             if self.perm_vec:
                 q_tensor = tf.transpose(q_tensor, perm=self.perm_vec)
         else:
             q_tensor = tf.quantization.fake_quant_with_min_max_vars(inputs, _min, _max,
-                                                                    num_bits=self.quantization_config.weights_n_bits)
+                                                                    num_bits=self.num_bits)
 
         return q_tensor
 
 
+    def convert2inferable(self) -> qi.BaseKerasInferableQuantizer:
+        """
+        Convert quantizer to inferable quantizer.
+
+        Returns:
+            BaseKerasInferableQuantizer object.
+        """
+        if self.power_of_two:
+            pot_threshold = 2**np.ceil(np.log2(self.quantizer_parameters[THRESHOLD_TENSOR]))
```

Comment:
Why do we change the threshold? Isn't it supposed to be POT already?

Reply:
That's true if the user trains PoT quantizers threshold in a somewhat naive way... Maybe instead just raise and exception if the threshold aren't PoT? Or at least a warning. @haihabi what do you think?

Reply:
I think this exception should be handled inside the InferableQuantizer